tests/test_device.py

```diff
@@ -155,8 +156,8 @@ def test_query_gid_ex_bad_flow(self):
                              f'Got {os.strerror(ex.error_code)} but '
                              f'Expected {os.strerror(errno.ENODATA)}')
         else:
-            raise PyverbsRDMAError(f'Successfully queried '
-                                   'non-existent gid index {i}')
+            raise PyverbsRDMAError('Successfully queried '
+                                   f'non-existent gid index {no_gid_index}')
```

Comment:
Any chance of exception raised from 134 ~ 142?

Reply:
I don't think so.